### DISCUSSION TITLE
Different controllers Connection on Multiplay games

### DISCUSSION THREAD

**Discussion by alexgame8818** [darthcloud/BlueRetro#370] [Q&A]
Thank you for your project. This is very interesting.
I have bulit SNES,NGC,DC,N64,PS2,NES adaptors for 4 or 5 multiplayers.
But it's very hard to make PS5,PS4,wiiu pro,switch pro,switch joycon,XBOX ones controllers work at the same time in multiplay games.
When PS5 worked, other controllers would hardly connect to BlueRetro. 
When wiiu pro and switch pro contrllers conneted,others would not be able to connet to BlueRetro.
XBOX ONES controller is the hardest one to connect to RetroBlue(above 2 minutes ).
And I also have 10 pieces ESP32-DEVKITC which came to the same result.
Snes 5 multiplayers game can only be conneted by 3 players(SNES plug with 7 cables).
Mighty these ESP32-DEVKITCs have some problems,or too many kinds of controllers to be used at the same time because of the interference from each other?

**Comment by darthcloud** [comment]
YMMV, for over 2 players its best to use same controller type  PS4/PS5 being best ones.

That said I very often use 2x PS4, one switch and one wii u.

Xbox controller are particular and takes long time to pair.

Don't use more than one switch controller as they currently step on each other due to some bug.

**Reply by alexgame8818** [reply]
Thank you for your prompt.
But ,even have some consoles, only one or two controllers for one console.
I will prepare more PS4 controllers for multiplay.
And hope BlueRetro can support more type of controllers at the same time in the future.
Such as using a usb port to lock the connection after calbe linked.
Very Appreciate for your amazing project.
